functional-test/src/main/java/org/zanata/page/WebDriverFactory.java

```diff
@@ -107,11 +108,17 @@ private WebDriver createPlainDriver() {
     }
 
     private WebDriver configureChromeDriver() {
+        // TODO can we use this? it will use less code, but it will use DISPLAY rather than webdriver.display
+//        System.setProperty(ChromeDriverService.CHROME_DRIVER_LOG_PROPERTY,
```

Comment:
On 2014-12-22 14:03, Damian Jansen wrote:
> In functional-test/src/main/java/org/zanata/page/WebDriverFactory.java
> <https://github.com/zanata/zanata-server/pull/655#discussion-diff-22153985>:
> 
>> @@ -107,11 +108,17 @@ private WebDriver createPlainDriver() {
>>      }
>>  
>>      private WebDriver configureChromeDriver() {
>> +        // TODO can we use this? it will use less code, but it will use DISPLAY rather than webdriver.display
>> +//        System.setProperty(ChromeDriverService.CHROME_DRIVER_LOG_PROPERTY,
> 
> With also
> 
>     System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY,
>     getChromeDriver().absolutePath));
> 
> yes?
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/655/files#r22153985>.
> 

I think you're right.


-- 
Sean Flanigan

Senior Software Engineer
Engineering - Internationalisation
Red Hat